.travis.yml

```diff
@@ -1,6 +1,8 @@
 language: objective-c
 xcode_project: SkyFloatingLabelTextField/SkyFloatingLabelTextField.xcodeproj
 xcode_scheme: SkyFloatingLabelTextField
+before_install:
+    - gem install bundler
```

Comment:
Why is this needed? It should be handled by Travis already I believe

Reply:
This is because bundler is out to date in some of virtual machines. As here, for example: https://drive.google.com/file/d/10s3ZcLZz4kbYsNdCIdaQagPY1NqH9jcG/view?usp=sharing

The error message would be this one: https://drive.google.com/file/d/1IP429lcqvnJO-2vsPYJolQHbOeTVaar7/view?usp=sharing